sdk/eventgrid/eventgrid/package.json

```diff
@@ -87,7 +87,9 @@
     "tslib": "^2.0.0"
   },
   "devDependencies": {
+    "@azure/dev-tool": "^1.0.0",
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
+    "@azure/service-bus": "^7.0.0",
```

Comment:
I felt a little weird doing this, but I couldn't really see a better way.  We need access to the service-bus package for our samples, because event grid delivers messages to services like service-bus and you consume them from there, so our consume samples need to use SDKs from other azure services to show the full scenario.  If there's something different/better I can be doing, please let me know.

Reply:
This is appropriate and fine as long as we can keep a handle on preventing these inter-package dependencies from becoming cyclic across the repo. To me this says "service-bus is required to build the code in this package. The sample code is in the package, so it's true.